Tests/LoggingTests/TestLogger.swift

```diff
@@ -292,3 +300,14 @@ internal struct TestLibrary {
         }
     }
 }
+
+#if compiler(>=5.3)
+internal func fullFilePath(_ filePath: StaticString = #filePath) -> StaticString {
```

Comment:
Why do we need this one in tests?
If #file == #filePath but just the warning around `= #file` sounds like there should be no difference on how we treat this in tests and the lib itself?